libdevcore/CommonIO.cpp

```diff
@@ -121,7 +121,10 @@ string dev::contentsString(std::string const& _file)
 
 void dev::writeFile(std::string const& _file, bytesConstRef _data)
 {
-	ofstream(_file, ios::trunc|ios::binary).write((char const*)_data.data(), _data.size());
+	ofstream s(_file, ios::trunc | ios::binary);
+	s.write(reinterpret_cast<char const*>(_data.data()), _data.size());
+	if (!s)
```

Comment:
shouldn't that be checked before trying to write to the file?
Additionally using a try/catch block?